dot/parachain/backing/statement_table.go

```diff
@@ -9,7 +9,7 @@ import parachaintypes "github.com/ChainSafe/gossamer/dot/parachain/types"
 type Table interface {
 	getCandidate(parachaintypes.CandidateHash) (*parachaintypes.CommittedCandidateReceipt, error)
 	importStatement(*TableContext, SignedFullStatementWithPVD) (*Summary, error)
-	attestedCandidate(*parachaintypes.CandidateHash, *TableContext) (*AttestedCandidate, error)
+	attestedCandidate(parachaintypes.CandidateHash, *TableContext) (*AttestedCandidate, error)
```

Comment:
can you check and tell why `parachaintypes.CandidateHash` to pointer?

Reply:
We will not modify the value of this variable. We don't need a pointer here.
What's your POV? Should we use a pointer?